database/elasticSearch.js

```diff
@@ -31,47 +34,49 @@ var randomGame = function(name) {
   var randomPrice = prices[Math.floor(Math.random() * prices.length)];
   var randomNumberOfOwners = Math.floor(Math.random() * 1000000);
 
-  var randomGame = [name, randomPublisher, randomReleaseDate, randomGenre, randomPrice, randomNumberOfOwners];
+  var randomGame = {
+    'name': name,
+    'publisher': randomPublisher,
+    'release_date': randomReleaseDate,
+    'genre': randomGenre,
+    'price': randomPrice,
+    'number_of_owners': randomNumberOfOwners
+  };
   return randomGame;
 };
 
-var generateRandomGames_Chunks = function(chunkNumber) {
-  var gamesArray = [];
+var generateRandomGames = function() {
+  var bodyArray = [];
   var currentGameName = null;
-  var currentGame = null;
-  var startIndex = (20000 * chunkNumber) + 1;
-  var endIndex = startIndex + 20000;
 
-  for (var i = startIndex; i < endIndex; i++) {
-    currentGameName = 'game #' + i;
-    currentGame = randomGame(currentGameName);
-    gamesArray.push(currentGame);
-  }
-  return db.addGames(gamesArray);
-};
+  // var startIndex = (20000 * chunkNumber) + 1;
+  // var endIndex = startIndex + 20000;
 
-var generateChunks = function(number) {
-  console.log(v8.getHeapStatistics());
-  var promises = [];
-  for (var i = 1; i <= number; i++) {
-    var promise = generateRandomGames_Chunks(i);
-    promises.push(promise);
+  for (var i = 1800001; i <= 2200000; i++) {
```

Comment:
Put these numbers in a const variable so that it's more clear about what they are used for.